### ISSUE TITLE
fine tuning pwc trained model

### ISSUE THREAD

**Issue by yoryory** [simonmeister/UnFlow#66]
hey,
can i use your training code to fine tuning not flownet model?
in my case i went fine tuning my trained pwc model on my data
thanks

**Comment by simonmeister** [comment]
Depending on training mode, you would need to adapt some of the code in

https://github.com/simonmeister/UnFlow/blob/master/src/e2eflow/core/supervised.py 
or
https://github.com/simonmeister/UnFlow/blob/master/src/e2eflow/core/unsupervised.py

All of the losses should work fine regardless of network and if I remember correctly the rest of the code outside of supervised.py and unsupervised.py shouldn't depend on the network. I may be wrong and there could be other changes required, but they would probably be minor.

**Comment by yoryory** [comment]
thank you for your respond
